source/includes/fact-query-converter-failure-errors.rst

```diff
@@ -0,0 +1,2 @@
+If the query converter has errors, you can view the details under 
```

Comment:
[suggestion] We have two prepositions here. Did you want _under_ or _in_?

Reply:
in